docs/userguides/departingemployee.md

```diff
@@ -1,6 +1,8 @@
-# Add or Remove users from the Departing Employee List
+# Add or Remove Users From the Departing Employees List
```

Comment:
Departing Employees List sounds like a list of lists to me

Reply:
hmm. "list" is the term we're using in IX documentation (instead of "lens"). it's a list of employees/users.

Reply:
The "S" on Employees is what I was referring to. If this how normal talk, then I am ok with it.

Reply:
An "employee list" is also a list of employees though. The s is implied by the "list", isn't it?

Reply:
Consistency is good! Best to match the IX docs.